H/HDF5/Deps.toml

```diff
@@ -1,12 +1,12 @@
-["0.10-0.11"]
+["0.10-0.11.0"]
```

Comment:
Is this change expected?

Reply:
Yes, you don't depend on the listed packages.

Reply:
I see that makes sense, since they are only test deps, not package deps.